### PR TITLE
11/23 issues

### PR DESCRIPTION
**Pull request by aspiece**

### CODE REVIEW

**Comment by aspiece** [comment]
Added Rachel and Dan. As soon as I someone can review this, it can get merged. I will be closing a bunch of issues, so more will be coming.

File: docs/lab_42.md.html [hunk 1]
```diff
@@ -46,21 +60,27 @@
...
-| 3.1 Add nouns                                          | 0.25     |
-| 3.2 Add other 5 parts of speech                        | 0.25     |
-| 3.3 Remove a part of speech                            | 0.5      |
-| **Total**                                      | **2.0** |
+| **Criteria** | **Points** |
+| :------ | :-- |
+| 1.1 6 lists | 0.5    |
+| 2.1 Report block | .25 |
+| 2.2 Reporter blocks | .75 |
+| 3.1 sentence script  | 0.5 |
```
Comment (Dan-TEALS) [inline docs/lab_42.md.html:78]: Inconsistent capitalization.

**Review by Dan-TEALS — CHANGES_REQUESTED** [review]
A few comments that should be corrected before merging. PowerPoint slides not reviewed.

**Comment by aspiece** [comment]
@Dan-TEALS  sorry, needed to add that and was still in this branch. Just added the resources from meetup #2.

**Pull request merged**